src/components/Datasets.js

```diff
@@ -188,22 +192,37 @@ class Datasets extends Component {
       return {
         key: id,
         name,
-        dataset: [
-          <Typography
-            className={buildDatasetsListNameClass(name)}
-            variant="subtitle1"
-            key="name"
-          >
-            {name}
-          </Typography>,
-          <Typography
-            className={buildDatasetsListDescriptionClass(name)}
-            variant="caption"
-            key="description"
-          >
-            {description}
-          </Typography>,
-        ],
+        dataset: (
+          <>
+            <Grid container alignItems="center" spacing={2}>
+              <Grid item>
+                <Typography
+                  className={buildDatasetsListNameClass(name)}
+                  variant="subtitle1"
+                  key="name"
+                >
+                  {name}
+                </Typography>
+              </Grid>
+              {schemaType !== SCHEMA_TYPES.NONE && (
+                <Grid item>
+                  <Chip
+                    size="small"
+                    color="primary"
```

Comment:
Parameterize this similar to the label, as other schemas won't have their `<Chip />` be the same purple color